OpenRefine/OpenRefine.download.recipe

```diff
@@ -3,13 +3,19 @@
 <plist version="1.0">
 <dict>
 	<key>Description</key>
-	<string>Downloads the current release of OpenRefine from Github.</string>
+	<string>Downloads the current release of OpenRefine from Github.
+	
+By default, "pre-releases" are included. Set INCLUDE_PRERELEASES
+to an empty value to only download releases on GitHub marked as
+"final."</string>
 	<key>Identifier</key>
 	<string>com.github.n8felton.download.OpenRefine</string>
 	<key>Input</key>
 	<dict>
 		<key>NAME</key>
 		<string>OpenRefine</string>
+                <key>INCLUDE_PRERELEASES</key>
+                <string>true</string>
```

Comment:
I know the previous recipe had this set to true by default, but do we still want that to be the default? I've always followed the policy that AutoPkg should always grab `Stable` by default, and `Beta` or `Dev` or `Pre-Release` by override.

Reply:
I'd prefer that it always grab Stable by default but I didn't want to step on others' setups. You won't here any complaints from me and I'd be happy to modify it as such 😄

Reply:
I vote stable too!